crud/src/main/java/com/redhat/lightblue/rest/crud/cmd/AbstractLockCommand.java

```diff
@@ -86,6 +90,7 @@ public static AbstractLockCommand getLockCommand(String request) {
 
     @Override
     public CallStatus run() {
+        RequestMetrics.Context context = metrics.startLockRequest(getLockCommandName(), domain);
```

Comment:
I don't think we have lock implementation at crud level. Everything lock related seems to be at rest level only. Same is the case with GenerateCommand. So this is the only place their metrics can be tracked.